### DISCUSSION TITLE
地形图加载

### DISCUSSION THREAD

**Discussion by 12123456** [zouyaoji/vue-cesium#530] [Q&A]
1、VcTerrainProviderTianditu  版本3.2.6（Cesium最新版本）
![image](https://github.com/zouyaoji/vue-cesium/assets/26050102/dea316ce-bd69-4a44-96f0-dc51b3e1de21)
官网示例报错，一直没找出原因，大佬看看能不能复现

2、VcTerrainProviderCesium 同上
![image](https://github.com/zouyaoji/vue-cesium/assets/26050102/4e1239d5-c6ac-45e2-b616-3fa0e59e739b)
accessToken 设置不成功

项目是fork的github上的 [VueCLI4 + Vue3Js + Vuex4 + VueRouter4 项目模板]
麻烦大佬看看可以从哪些方面排查一下

**Comment by zouyaoji** [comment]
1、文档里面有提到哈：
![image](https://github.com/zouyaoji/vue-cesium/assets/24430116/aa921e49-cbc5-41fb-8682-d490c413e37d)
2、看看是不是包裹了vc-config-provider组件，如果有这个会优先取这个的。

**Reply by 12123456** [reply]
ok，谢谢大佬
